src/app/proposals/icons/TokenIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import { RbtcIcon } from './rbtc'
```

Comment:
Can't we use TokenImage instead of this?

Reply:
oh... I initially tried to use a regular PNG. But the second icon is a simple SVG. One file is PNG and another file SVG. If I keep 2 different file types, I couldn't make them look similar... one was always bigger than the other and I could do nothing. Or the second file will be more pixelated than the other. This approach was the only one which worked, though may be it looks strange. (I tried different)

Reply:
Francis, I tried to utilize it, but
1) the icons are of different size there ; 
2) it breaks the whole page. See screenshots

<img width="237" height="47" alt="Screenshot 2025-07-18 at 02 20 29" src="https://github.com/user-attachments/assets/84e5ddbb-1510-420c-b2c9-ba6271b502d7" />
<img width="662" height="287" alt="Screenshot 2025-07-18 at 02 20 14" src="https://github.com/user-attachments/assets/24cbc920-0d8f-46bb-92a1-2603ead4f856" />

Reply:
<img width="241" height="55" alt="Screenshot 2025-07-18 at 02 33 40" src="https://github.com/user-attachments/assets/fdd67bae-91c6-4732-bc8a-1527049104be" />
Compare with mine. They are of the same size.